docs/components/dartpad/dividers/vertical_divider/main.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: MyStatelessWidget(),
+      ),
+    );
+  }
+}
+
+class MyStatelessWidget extends StatelessWidget {
+  MyStatelessWidget({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Row(
+        children: <Widget>[
+          Expanded(
+            child: Container(
+              color: Colors.amber,
```

Comment:
I'd suggest removing the colors, since it makes it harder to see the divider itself. And then regenerate the screenshots.

Reply:
Done